examples/browser-app/package.json

```
@@ -5,8 +5,8 @@
   "scripts": {
     "build": "theia build --mode development",
     "prepare": "yarn build",
-    "start": "theia start --WF_GLSP=5007 --root-dir=../workspace",
-    "start:debug": "theia start --WF_GLSP=5007  --root-dir=../workspace --loglevel=debug --debug",
+    "start": "theia start --WF_GLSP=0 --root-dir=../workspace",
+    "start:debug": "theia start --WF_GLSP=0  --root-dir=../workspace --loglevel=debug --debug",
```

The debug variant is used for server debugging. We need a fixed port to do that
```suggestion
    "start:debug": "theia start --WF_GLSP=5007  --root-dir=../workspace --loglevel=debug --debug",
```